docs/stdlib/datetime.rst

```diff
@@ -680,10 +682,12 @@ EdgeDB stores and outputs timezone-aware values in UTC.
                   std::datetime_get(dt: cal::local_datetime, \
                                     el: str) -> float64
 
+    Returns the element of a date/time type from the specified ``el``.
+
     Extract a specific element of input datetime by name.
 
-    The :eql:type:`datetime` scalar has the following elements
-    available for extraction:
+    Only the :eql:type:`datetime` type has the following elements available
```

Comment:
I believe all of these units are available to use with either of data types supported by this function. Unless I'm mistaken:
```suggestion
    You may pass any of these unit names for *el*:
```